cmd/dcrdata/views/agendas.tmpl

```diff
@@ -104,137 +99,136 @@
-                                <span class="text-secondary fs14">quorum achieved</span>
-                              {{else}}
-                                <span class="text-secondary fs14"><span class="fw-bold">{{.VoteCount}} of {{.Quorum}}</span> votes cast</span>
-                              {{end}}
-                            </div>
-                            <div class="d-inline-block text-center p-2">
-                              <span class="fs16 fw-bold">Approval</span><br>
-                              <div
-                                class="meter text-center js-only large-gap"
-                                data-target="agendas.approvalMeter"
-                                data-approval="{{.Approval}}"
-                                data-threshold="{{.PassThreshold}}"
-                                >
-                                {{printf "%.1f" (f32x100 .Approval)}}%
-                              </div><br>
-                              <span class="text-secondary fs14">{{printf "%.0f" (f32x100 .PassThreshold)}}% passes,
-                                &lt; {{printf "%.0f" (f32x100 .FailThreshold)}}% rejects, <br>
-                                otherwise vote again
-                              </span>
-                            </div>
-                          </div>
-                          <span></span>{{/* FLEX SPACER */}}
+                  {{if .IsVoting}}
+                    <div>
+                      {{if not .QuorumAchieved}}
+                        <div class="bg-white d-inline-block p-2 fs16 text-center mx-2 border"><span class="fw-bold">{{.ID}}</span> has not achieved quorum yet</div>
+                      {{else if .IsLocked}}
+                        <div class="bg-white d-inline-block p-2 fs16 text-center mx-2 border"><span class="fw-bold">{{.ID}}</span> has passed</div>
+                      {{else if .IsWinning}}
+                        <div class="bg-white d-inline-block p-2 fs16 text-center mx-2 border"><span class="fw-bold">{{.ID}}</span> is winning, but not a lock yet</div>
+                      {{else if .IsLosing}}
+                        <div class="bg-white d-inline-block p-2 fs16 text-center mx-2 border"><span class="fw-bold">{{.ID}}</span> is losing, but hasn't been rejected yet</div>
+                      {{else}}
+                        <div class="bg-white d-inline-block p-2 fs16 text-center mx-2 border">Voting on <span class="fw-bold">{{.ID}}</span></div>
+                      {{end}}
+                    </div>
+                    <div class="d-flex justify-content-around align-items-start flex-wrap mt-2">
+                      <div class="d-inline-block text-center p-2">
+                        <span class="fs16 fw-bold">Quorum</span><br>
+                        <div
+                          class="quorum meter text-center js-only large-gap"
+                          data-target="agendas.quorumMeter"
```

Comment:
The problem was that these targets are not usable since this is in a gohtml template **loop** and it makes several identically-named targets.  I may as well remove this line.  The controller that initializes the page now just uses `querySelectorAll` and does a `forEach` through all of these meters, which now have an added class to identify them.